test/extended/testdata/ldap/ldapserver-deploymentconfig.yaml

```diff
@@ -0,0 +1,79 @@
+---
+kind: DeploymentConfig
+apiVersion: apps.openshift.io/v1
+metadata:
+  name: openldap-server
+  creationTimestamp: 
+spec:
+  strategy:
+    type: Recreate
+  triggers:
+  - type: ConfigChange
+  replicas: 1
+  selector:
+    name: openldap-server
+  template:
+    metadata:
+      creationTimestamp: 
+      labels:
+        name: openldap-server
+    spec:
+      serviceAccountName: ldap
+      containers:
+      - name: openldap-server
+        # NOTE: This image is built from the images/openldap directory. Temporary repo location.
+        # It provides a default initialization of an openldap server. LDIF data is in /usr/local/etc/openldap/init.ldif
+        # TODO: mount in ldif data here rather than built-in
+        image: docker.io/mrogers950/origin-openldap-test:upstream
+        ports:
+        - containerPort: 389
+          protocol: TCP
+        - containerPort: 636
+          protocol: TCP
+        terminationMessagePath: "/dev/termination-log"
+        imagePullPolicy: Always
+        readinessProbe:
+          exec:
+            command:
+            - ldapsearch
+            - "-x"
+            - "-b"
+            - dc=example,dc=com
+          initialDelaySeconds: 5
+          timeoutSeconds: 1
+        volumeMounts:
+        - name: ldap-config
+          mountPath: /usr/local/etc/ldapconf
+        - name: scripts
+          mountPath: /usr/local/bin
+        - name: ldap-cert
+          mountPath: /usr/local/etc/ldapcert
+      volumes:
+      - name: ldap-cert
+        secret:
+          secretName: ldap-cert
+      - name: ldap-config
+        configMap:
+          name: ldap-config
+          items:
```

Comment:
I don't mind this, but I also don't mind loading it all and sorting it out later.